### PR TITLE
IMDB Watchlist importer

### PR DESCRIPTION
**Pull request by lechat**
Added ability to watch for IMDB Watchlist (Issue #146) The code is based on Trakt watchlist importer.

It operates using URL of the "Export this list" link, which is displayed at the bottom of each list page. I added configuration section under "Extras" tab for it.

### CODE REVIEW

File: app/views/config/index.html [hunk 1]
```diff
@@ -443,6 +443,14 @@
...
 		</div>
 
+		<div class="group imdb">
+			<fieldset class="inlineLabels">
+				<div>Got a jailbroken iPhone? Get the CouchPotato app on Cydia! <span>(search for "<strong>iCouchPotato</strong>")</span></div>
+				<img src="${baseUrl}media/images/iphone1.png" /><img src="${baseUrl}media/images/iphone2.png" /><img src="${baseUrl}media/images/iphone3.png" />
+				<div>Why isn't it in the normal Appstore you ask? Because Apple is being a bitch!</div>
+			</fieldset>
+		</div>
+
```
Comment (RuudBurger) [inline app/views/config/index.html]: Why is this?

**Comment by RuudBurger** [comment]
Looking good, but you need to add the enabled to https://github.com/RuudBurger/CouchPotato/blob/master/app/controllers/config.py#L63
And there is some code that isn't needed (see other comment)

**Comment by lechat** [comment]
Yes, the extra section was unnecessary and was a part of my try outs. Thanks for noticing it! Code review FTW! :) 

I removed it and added missing bit.

**Comment by RuudBurger** [comment]
Thanks for your work. WIll use this in the new build too, thanks!

**Pull request merged**